components/site/site.css

```diff
@@ -0,0 +1,289 @@
+/*!
+Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+.spectrum-Site {
+	display: flex;
+	flex-direction: column;
+
+	block-size: 100vh;
+	inline-size: 100vw;
+
+	color: var(--spectrum-body-color);
+
+	background-color: var(--spectrum-background-layer-2-color);
+}
+
+.spectrum-Site-content {
+	display: grid;
```

Comment:
With the change to `grid` from `flex`, the `.spectrum-Site-nav` is no longer scrolling. I think we need to either switch this back to `flex` or we'll need to add some additional styles to bring back the scrolling.

Reply:
Can you share more detail about where you don't see the scroll?  The scroll is happening in my desktop view on the navigation content but with the goal of keeping the icon and search bar always in view.